lib/input_file_converter.py

```diff
@@ -479,7 +479,9 @@ def parse_exon_file(self, input_file):
         exon_file = input_file.replace('_protein.fa', '.exons.txt')
         five_prime_positions, three_prime_positions = [], []
         with open(exon_file, 'r') as fh:
-            reader = csv.DictReader(fh, delimiter='\t')
+            dialect = csv.Sniffer().sniff(fh.read())
+            fh.seek(0)
+            reader = csv.DictReader(fh, delimiter=dialect.delimiter)
```

Comment:
Would it make sense to pass `dialect=dialect` here instead of pulling out only the delimiter?  Guess it depends how much we trust the sniffer :smile: